### ISSUE TITLE
Refactor test to Node.js | Either Mocha or Jest

### ISSUE THREAD

**Issue by laugekj** [MadsHogenhaug/MiniTwit#2]
Tests should be refactored to similar node framework. Either Mocha or Jest most likely. Both are good and one is made from Facebook, so that has the stamp of approval.

**Comment by Marcus-Friis** [comment]
Major race condition issue in all tests with regards to instantiation of sqlite database:
"Error: Error: SQLITE_ERROR: no such table: user"

**Comment by Marcus-Friis** [comment]
Issue should be solved with [these changes](#15 ). 
The essence of the change is changing the `init_DB` function to be asynchronous.
```javascript
async function init_DB() {
    try {
        let db = connect_DB();
        const schema = await fs.readFile(SCHEMA_FILE_PATH, 'utf-8');

        await new Promise((resolve, reject) => {
            db.exec(schema, (error) => {
                if (error) {
                    reject(error);
                } else {
                    resolve();
                }
                db.close();
            });
        });
    } catch (error) {
        console.log(error);
    }
}
```
